containers/anndata-to-ui/create-test-input.py

```diff
@@ -38,10 +38,12 @@ def create_h5ad_secondary_analysis(h5ad_path):
 
 
 def create_h5ad_scvelo(h5ad_path):
-    data = array(
+    # scvelo uses spliced inputs:
```

Comment:
Yes and no -- scVelo uses both spliced and unspliced counts, separately, to compute RNA velocity and other measures. I probably should rename that parameter value; that was to remind myself that scVelo accepts the H5AD file that has separate layers for spliced and unspliced counts, while I was putting the pieces of the pipeline together a while ago.